### DISCUSSION TITLE
Inconsistency between zeus-cli and VS Code

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#219] [Developer Feedback]
Hello.
I've a project in VS code and I'm able to build it with VSCode plug-in. But same project can't be built using zeus-cli.

![Screenshot from 2023-07-18 15-36-28](https://github.com/zepp-health/zeppos-forum/assets/5894573/4fc01bd4-4ab9-4805-8ffc-41feeb443122)
Error message isn't clear and doesn't give any clues what should be fixed.

part from app.json
`"targets": {
        "466x466-gtr-4": {
            "module": {
                "watchface": {
                    "path": "watchface/index",
                    "main": 1,
                    "editable": 0,
                    "lockscreen": 0,
                    "photoscreen": 0
                }
            },
            "platforms": [
                {
                    "name": "gtr4",
                    "deviceSource": 7930112
                },
                {
                    "name": "gtr4-w",
                    "deviceSource": 7930113
                },
                {
                    "name": "Amazfit GTR 4 Limited Edition",
                    "deviceSource": 7864576
                },
                {
                    "name": "Amazfit GTR 4 Limited Edition",
                    "deviceSource": 7864577
                }
            ],
            "designWidth": 466
        }
    },`

**Comment by htoooth** [comment]
hi @kartun83 
Which cli version do you use?

It would be helpful if you could provide a minimal project that reproduces the issue, so that we can troubleshoot the problem more effectively.

**Reply by kartun83** [reply]
Hello.
Currently with 1.3.0-beta.10, but reproducable with stable as well.
[zeus_unsupported_target.zip](https://github.com/zepp-health/zeppos-forum/files/12090803/zeus_unsupported_target.zip)

**Comment by zeppos-hx** [comment]
There is no watch with the deviceSource value `7864576`, you should remove it from app.json

**Reply by kartun83** [reply]
This app.json was exported from Watchface maker.  So, either there is an error in Watchface maker, or you are wrong and zeus-cli templates are outdated.
Anyway, the error message isn't helpful and doesn't provide any clue how to fix the error.
Plugin for VS Code is able to build this project without errors. Why zeus-cli threat this as error and not as warning ?
